build/configs/rtl8721csm/hello/Make.defs

```diff
@@ -54,7 +54,16 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
+ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
+LDSCRIPT = rlx8721d_img2_ns.ld
```

Comment:
How about adding/merging line 64 and 65 in here?

Reply:
You mean merge to "LINKLIBS=${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a", right?
I planning to merge the "ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)" as well, is that okay?

Reply:
This is what I want to. Sorry to confuse you.